spec/unit/software_spec.rb

```diff
@@ -190,8 +190,8 @@ module Omnibus
         end
       end
 
-      context "on macOS" do
-        before { stub_ohai(platform: "mac_os_x") }
```

Comment:
Just FYI you don't need to specify the version here and if you leave it out then we don't need to update things in the future.